andela/Healthchecks_Asgardians
# Ft nag notification 145028381

#### What does this PR do?
As a user I want to be able configure nag interval times
As a user I want to receive an email when the nag time has elapsed
As a user I wanta view on the dashboard, a check on nag mode with a nag icon

#### Description of Task to be completed?

GIVEN one of the User's jobs has gone down and he has been notified
WHEN he does not resolve the issue within a specified period
THEN he should be notified again and again till he caters to the failed job

#### How should this be manually tested?

clone the repository
cd into the repository
run the command ./manage.py runserver
-start server and run the app
-create a check and set the period, grace and nag time
open a new terminal
run the command python sendalerts.py
